CGPs/0024.md

```diff
@@ -8,7 +8,7 @@
 
 ## Overview
 
-This is the last proposal unfreezing and fully activating cEUR. It is to be executed after CGP #22, CGP #23, CGP #25 and CGP #26 were executed and CELO/EUR oracles are deployed and reporting. It adds ExchangeEUR and StableTokenEUR to the reserve and enables cEUR as a gas currency.
+This is the last proposal unfreezing and fully activating cEUR. It is to be executed after [CGP #22](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md), [CGP #23](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0023.md), [CGP #25](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0025.md) were executed and CELO/EUR oracles are deployed and reporting. It adds ExchangeEUR and StableTokenEUR to the reserve and enables cEUR as a gas currency.
 
 ## Proposed Changes
 
```

Comment:
Nit: I know this already existed but `Fill out the following template for each transaction in the proposal` should probably be removed

Reply:
Removed.